README.md

```diff
@@ -13,8 +13,8 @@ the Bitcoin Core JSON-RPC APIs as rust functions.
 the json responses from the Bitcoin Core JSON-RPC APIs. bitcoincore-rpc depends on this.
 
 # Usage
-Given below is an example of how to connect to the Bitcoin Core JSON-RPC for a Bitcoin Core node running on `localhost`
-and print out the hash of the latest block.
+Given below is an example of how to connect to the Bitcoin Core JSON-RPC for a Bitcoin Core node running on `localhost` 
+and print out the hash of the latest block. 
```

Comment:
It looks like you've added trailing whitespace here. Can you instead remove all the existing trailing whitespace?